bidwire/notice.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime, timedelta
+from sqlalchemy import Column, Integer, String, DateTime, func, Text, JSON
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import validates
+from enum import Enum
+
+Base = declarative_base()
+
+
+class Notice(Base):
```

Comment:
Let's delete this file and all other mentions of notices. (If we ever want to bring them back, we can get all your work back from the commits in this PR.)